### PR TITLE
💄(funcorporate) fix course category badge layout

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose
On funcorporate, there was a difference between mockups and integration on course detail page; category badges layout was not correct.

## Proposal

- [x]  Extend subheader styles to customize category badges displayed inside it.

Capture d’écran 2020-09-11 à 10 16 18

### CODE REVIEW

**Comment by sampaccoud** [comment]
@jbpenrath Do you think it is possible to do this with css overrides only? If necessary, we can add things on richie side to make it possible. We should avoid maintaining template overrides for this kind of theme customisation.

**Comment by jbpenrath** [comment]
> @jbpenrath Do you think it is possible to do this with css overrides only? If necessary, we can add things on richie side to make it possible. We should avoid maintaining template overrides for this kind of theme customisation.

Ok, I will overrides css instead of overrides template.

**Comment by jbpenrath** [comment]
@sampaccoud Job done! 🙂

File: sites/funcorporate/CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Customize social networks links
 
 ### Fixed
-
+- Update `course_detail` template to fit fun-corporate mockups.
```
Comment (sampaccoud) [inline sites/funcorporate/CHANGELOG.md:20]:
```suggestion
- Override category badge style to fit fun-corporate mockups.
```

**Review by sampaccoud — APPROVED** [review]

File: sites/funcorporate/CHANGELOG.md (hunk 1 above)
Comment (jbpenrath) [reply sites/funcorporate/CHANGELOG.md:20]: J'y ai pas pushé 😅

Comment (jbpenrath) [reply sites/funcorporate/CHANGELOG.md]: Whoops I forgot to force push..

**Pull request merged**